### PR TITLE
ADD branches cleanup script

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: -we sometimes leave merged branches around-

## What I did
I added a script to remove them

## How to test
Run script:
```sh
./scripts/clean-merged-branches.sh
```
Will list the branches

Run script:
```sh
./scripts/clean-merged-branches.sh -f
```
Will remove listed branches from remote

### CODE REVIEW

**Comment by UsulPro** [comment]
tried to test it. always get `no merged branches detected`

**Comment by ndelangen** [comment]
Yeah, because i already removed all the merged branches using this 😃

**Comment by UsulPro** [comment]
Great script, Norbert!

I noticed that it missed the branch `remotes/origin/Gongreg/master`, but we can remove it manually :)
I added two `test/*` branches (one is merged to another) and it missed them as well, but I can detect them by `git branch -a --merged` so not sure how to handle such cases

**Comment by Hypnosphi** [comment]
It doesn't work if the origin is named other than `origin` (in my case  it's `upstream`). I think I can rename though

File: scripts/clean-merged-branches.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+# fetch and prune remote branches
+git fetch origin --prune
+
+# grab list of merged branches
+branches=$(
+  git branch -a --merged origin/master |
+  grep remotes/origin/ |
+  grep -v /master |
```
Comment (Hypnosphi) [inline scripts/clean-merged-branches.sh]: This excludes `remotes/origin/Gongreg/master` as well

Comment (Hypnosphi) [reply scripts/clean-merged-branches.sh]: I will fix that and add a confirm prompt instead of "run this command with `-f`" message

**Comment by Hypnosphi** [comment]
I changed "Run with '-f' if you're sure" to "Press 'y' if you're sure"

**Review by usulpro — APPROVED** [review]
LGTM

**Pull request merged**